### DISCUSSION TITLE
Proposing changes in products

### DISCUSSION THREAD

**Discussion by MSfastcr** [allegro/allegro-api#9746] [Suggestions]
Hello,
We would like to be able to make product suggestions via API about our products. We used to make such suggestions using Allegro sellers portal, but this option become unavailible, attepts result in error (see screenshot, it says we can not change the product). Our Allegro contact also told us that API option (POST /sale/products/{productId}/change-proposals) will also become unavailible. How should we proceed as a owner of the brand? We would like to correct some mistakes in Allegro catalog, but we are not able to do so.

Thank you for your answer

Sincerely
FAST ČR

![image](https://github.com/user-attachments/assets/56abecc9-a6c7-425f-aeb9-7c3b5544065c)

**Comment by MartaNowaczyk** [comment]
If something is not working on the website, please contact us through the [contact form](https://allegro.pl/pomoc/kontakt). Discussions is the place where you can make suggestions, but regarding the Allegro API.

**Comment by jezewskiBNetrox** [comment]
@MSfastcr You should ask through contact form as Marta (actual Allegro worker) said. This GitHub channel of help is only for API alone, not for general consulting. 

From my knowledge however some products are blocked for any modification by owner of rights to product. In such case this is impossible to make any modifications to the product by seller.
